recipes/dedupe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "dedupe" %}
+{% set version = "2.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dedupe-{{ version }}.tar.gz
+  sha256: f360c5a3ac940623f00971c93836213147b826b7f87bbb81ce9e7ff73c67847a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
Just curious, why are you skipping Windows?

Reply:
dedupe is dependent on pyhacrf-datamade which is dependent on pylbfgs which is not available on Windows.  Since this is not noarch Python, I can just skip the build on Windows.

Reply:
That is fine. You can remove the skip. When a user tries to install they'll get a message about the missing dep.

Reply:
Oh. This one is arch'ed. Then we do need the skip.